lib/vscode/build/lib/test/i18n.test.ts

```diff
@@ -6,7 +6,7 @@
 import assert = require('assert');
 import i18n = require('../i18n');
 
-suite('XLF Parser Tests', () => {
+describe('XLF Parser Tests', () => {
```

Comment:
After removing `mocha` I had to convert this to `describe` because jest doesn't have `suite` as a function.